### ISSUE TITLE
Packet forward middleware doesn't work

### ISSUE THREAD

**Issue by Ehsan-saradar** [cosmos/gaia#1441]
## Summary of Bug

Sending transfer packets with packet forward middleware receiver field (https://github.com/strangelove-ventures/packet-forward-middleware#example) format returns unsuccessful acknowledgment and it will not relay the transfer to the desired chain. (I tested this in dev environment not testnet or mainnet)

I think the issue is with this line https://github.com/cosmos/gaia/blob/v7.0.1/app/app.go#L440
Instead of passing `transferIBCModule` itself to `ibcRouter` it should be replaced by `routerModule`

## Version

gaiad version: v7.0.1

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned
- [ ] Is a spike necessary to map out how the issue should be approached?

**Comment by mmulji-ic** [comment]
Thanks @Ehsan-saradar for opening the ticket. This is a known issue and there is a fix that needs to be applied from StrangeLove. It was not possible to get that fix into v7.0.0, because it missed the time window and because of the state breaking nature, we can't include it in point releases. Therefore this is scheduled to be fixed for the Rho update in May/June.

**Comment by mmulji-ic** [comment]
@mmulji-ic ping Strangelove folks that we've got an IBC beta.

**Comment by jtieri** [comment]
> @mmulji-ic ping Strangelove folks that we've got an IBC beta.

Hey Milan, just want to confirm that the plan forward for the rho release is to use SDK v0.46.0 and ibc-go v5 that is being replaced with a tagged version from Notional's repo? 
e.g. `github.com/cosmos/ibc-go/v5 => github.com/notional-labs/ibc-go/v5 v5.0.0-20220728082452-6f2e6df135b4`

**Comment by mmulji-ic** [comment]
Hi @jtieri were using the Notional version for a mo, @faddat has completed an SDK upgrade for IBC as well. Carlos is in the process of making an RC for IBC any moment now.

**Comment by jtieri** [comment]
Thanks, that makes more sense. I will update the packet-forward-middleware to the new dependency versions as well as Go 1.18.

Edit: I have updated the repo for the most part, I will hold off on review and merging until the IBC team cuts the v5 release.
https://github.com/strangelove-ventures/packet-forward-middleware/pull/30

We did create a test in `ibctest` for the packet-forward-middleware as well. It will need to be updated to work with the new release but perhaps it is something of interest to the team. 
https://github.com/strangelove-ventures/ibctest/blob/main/examples/packet_forward_test.go
